geomaster/scripts/render_mesh.py

```diff
@@ -0,0 +1,218 @@
+import os
+import torch
+import argparse
+import numpy as np
+import trimesh
+from geomaster.utils.remesh_utils import calc_vertex_normals
+from geomaster.utils.camera_utils import make_round_views
+from scipy.spatial.transform import Rotation
+import multiprocessing as mp
+from functools import partial
+import torch.multiprocessing as tmp
+import queue
+import time
+import nvdiffrast.torch as dr
+from matplotlib import image
+
+
+# ---- Config ----
```

Comment:
Configuration values should be defined as parameters rather than hardcoded directly in the code.